hw02-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,18 @@
+package homework;
+
+import java.util.LinkedList;
+
+//@SuppressWarnings({"java:S1186", "java:S1135", "java:S1172"}) // при выполнении ДЗ эту аннотацию надо удалить
+public class CustomerReverseOrder {
+
+    // todo: 2. надо реализовать методы этого класса
+    // надо подобрать подходящую структуру данных, тогда решение будет в "две строчки"
+    private final LinkedList<Customer> list = new LinkedList<>();
```

Comment:
LinkedList правильно.
Слева лучше указывать "минимальный" интерфейс.
```suggestion
    private final Deque<Customer> list = new LinkedList<>();
```